psi4/src/psi4/libmints/electrostatic.cc

```diff
@@ -62,6 +63,15 @@ void ElectrostaticInt::compute(SharedMatrix& result, const Vector3& C) {
     OneBodyAOInt::compute(result);
 }
 
+void ElectrostaticInt::set_origin(const Vector3& _origin) {
+    origin_ = _origin;
+    Zxyz_vector pcs;
+    // l2 includes the electron charge, legacy psi4 code does not, so
+    // we adopt this behavior here with -1.0
+    pcs.push_back({-1.0, {origin_[0], origin_[1], origin_[2]}});
+    engine0_->set_params(pcs);
```

Comment:
good catch!

Reply:
Credit goes to @maxscheurer for that, I borrowed it from him [here](https://github.com/psi4/psi4/pull/2473/files#diff-afe4b5648da445aad249f05fa847be3a9e29f5bc0051d2ae3a2e42531c11152dR56-R65)